pkgs/top-level/all-packages.nix

```diff
@@ -2271,11 +2271,12 @@ in
 
   ised = callPackage ../tools/misc/ised {};
 
-  isl = isl_0_15;
+  isl = isl_0_17;
   isl_0_11 = callPackage ../development/libraries/isl/0.11.1.nix { };
   isl_0_12 = callPackage ../development/libraries/isl/0.12.2.nix { };
```

Comment:
GCC and LLVM are the main users, and particular compiler versions need particular isl versions IIRC.  Still, it's possible some of those isn't needed anymore.